WcaOnRails/app/views/admin/results/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Edit a result") %>
```

Comment:
Hum because *you* changed the way react components were created/include?

Well at least that's how I understood it worked from how it's done for the posts widget and the competition results ^^
Feel free to let me know if I'm missing something!

Reply:
Oh, I wasn't even aware that the `react-rails` gem got rid of the need for the packs altogether. I thought those were needed regardless of how you reference the individual components. But who am I to argue with simplicity :D